docs/map_state/duplicates.md

```diff
@@ -0,0 +1,20 @@
+Atlas groups your data into semantically similar clusters powered by latent information contained in your embeddings.
+Under the hood, it utilizes an algorithm similar to [SemDeDup](https://arxiv.org/abs/2303.09540).
+
+
+You can access and operate on semantic duplicate clusters programmatically by using the `duplicates` attribute
+of an AtlasMap. 
+
+
+```python
+from nomic import AtlasProject
+
+map = AtlasProject(name='My Project').maps[0]
```

Comment:
super nitpick: In one of the tutorials, it may be good to showcase `get_map` instead of accessing the list of maps?